packages/remote-feature-flag-controller/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@metamask/remote-feature-flag-controller",
+  "version": "0.0.0",
+  "description": "Controller with caching, fallback, and privacy for managing feature flags via ClientConfigAPI",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/remote-feature-flag-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
```

Comment:
The preference for new open source packages is to use a dual MIT and Apache 2.0 license. See here for an example: https://github.com/MetaMask/design-tokens/blob/f8c5e466964a4709708bf35c3663bce6fa55c049/package.json#L17

And note the two license files

Reply:
Updated to use both licenses in 2bb0ade7adeb0ab3b390e46750a502f314de644d

Reply:
Not sure if this is the fix, a new `LICENSE` file is also added to severe both licenses ead62540f1253c5d4f5610663ec7bfbd1d0ddbdb